x/autopilot/types/parser.go

```diff
@@ -27,8 +27,10 @@ type ModuleRoutingInfo interface {
 
 // Packet metadata info specific to Stakeibc (e.g. 1-click liquid staking)
 type StakeibcPacketMetadata struct {
-	Action        string `json:"action"`
-	StrideAddress string `json:"stride_address"`
+	Action          string `json:"action"`
+	StrideAddress   string `json:"stride_address"`
+	IbcReceiver     string `json:"ibc_receiver,omitempty"`
+	TransferChannel string `json:"transfer_channel,omitempty"`
```

Comment:
if these are empty it's assumed that the token is returned to the original sender over the same source channel?